cves/CVE-2004-0811.yml

```diff
@@ -199,4 +201,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: this relied on arbitrary input via the ! operator
```

Comment:
Consider including the CWE mitigations to expand on explanation. Also, it looks like you should include a newline at the end of the file